Source/Core/Duality/Launcher/DualityLauncher.cs

```diff
@@ -1,22 +1,17 @@
 ﻿using System;
 using System.IO;
-using System.Linq;
 using Duality.Backend;
 using Duality.Resources;
 
 namespace Duality.Launcher
 {
 	public class DualityLauncher
 	{
-		public static void Run(string[] args)
+		public static void Run(LauncherArgs launcherArgs)
 		{
 			System.Threading.Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
 			System.Threading.Thread.CurrentThread.CurrentUICulture = System.Globalization.CultureInfo.InvariantCulture;
 
-			bool isDebugging = System.Diagnostics.Debugger.IsAttached || args.Contains(DualityApp.CmdArgDebug);
```

Comment:
Might be better to only set isDebugging to true if the actual parameter is passed.

Atm you will get different behavior when attaching the debugger depending if it happened before this line of code was executed or after. Better to just start the launcher with the debugging flag.

Reply:
Good idea, should make it more transparent how this behaves 👍